omod/src/main/java/org/openmrs/web/dwr/DWREncounterService.java

```diff
@@ -115,7 +116,9 @@ public Vector findBatchOfEncountersByPatient(String phrase, Integer patientId, b
 			} else {
 				objectList = new Vector<Object>(encs.size());
 				for (Encounter e : encs) {
-					objectList.add(new EncounterListItem(e));
+					EncounterListItem htmlSafeEncounter = new EncounterListItem(e);
+					htmlSafeEncounter.setEncounterType(Encode.forHtml(htmlSafeEncounter.getEncounterType()));
```

Comment:
looking good to me @annaowens  thanks, @isears  how does this look to you

Reply:
Yeah, I agree. I think this is ready to get merged in